doc/options.yaml

```diff
@@ -554,6 +554,14 @@ useALE:
   desc: >
     Use the arbitrary Lagrangian-Eulerian (ALE) formulation for unsteady mesh deformations.
 
+useExternalDynamicMesh:
+  desc: >
+    Use externally provided deformed mesh for the time spectral equation. Each time instance gets its own deformed grid. This can be used for both an aerodynamic case or an aeroelastic case.
+
+useTSInterpolatedGridvelocity:
+  desc: >
+    Use spectral differentiation to compute the grid velocity for the time spectral equation.
+
```

Comment:
Is there a scenario where you would want to use one of these options but not the other?

Reply:
A flexible grid must be run with the interpolated velocity. But a rigid grid can run with interpolated / analytic grid velocity.

Reply:
This makes sense, thanks